### PR TITLE
Changes to support higher contrast colors for accessibility

### PR DESCRIPTION
**Pull request by ddellacosta**
(...with some minor formatting tweaks as well.)

https://www.pivotaltracker.com/story/show/161903537

I am really not sure if I've chosen the right approaches here--so an honest and thorough critique will be most appreciated! In particular not sure about the way I've chosen to set the anchor link color globally.

### CODE REVIEW

File: src/sass/site.scss [hunk 1]
```diff
@@ -19,9 +19,21 @@ $border-radius-sm:            0px;
...
+ higher-contrast (compared to Bootstrap default of #007bff) for
+   accessibility
+
+// I don't know how to override this, and not sure what the best
+// approach is anyways (i.e. should I even try to reset the blue value
+// globally for this accessibility fix?)
+$blue: #006FE6;
+
+*/
+a { color: #006FE6; }
```
[the remark this answers is not in the document]
Comment (ddellacosta) [reply src/sass/site.scss]: Yeah, and more generally I wonder if there is a more appropriate way to specify this, even if we want it to be globally applied like this. This is where I'd really like @tadler 's feedback in particular.

[the remark this answers is not in the document]
Comment (tadler) [reply src/sass/site.scss]: This is okay! If we find that this alters the look of an anchor somewhere in a way we didn't intend, we can address that instance with styles written specifically for that anchor.

[the remark this answers is not in the document]
Comment (ddellacosta) [reply src/sass/site.scss]: Okay great, thanks Christine!

[the remark this answers is not in the document]
Comment (ddellacosta) [reply src/sass/site.scss]: @tank157 Let me know if Christine's response is satisfactory to you--I'll add, I am hoping that any weirdness here will shake out when we test it on staging.

[the remark this answers is not in the document]
Comment (tank157) [reply src/sass/site.scss]: Yup, it's fine with me. Probably the only way to track any issues with this down is in visual testing anyway

**Pull request merged**